src/wallet/bdk.rs

```diff
@@ -104,6 +135,33 @@ impl KeychainKind {
 
 const WATCH_ONLY_SWAPCOIN_LABEL: &str = "watchonly_swapcoin_label";
 
+/// The changes made to a wallet by applying an [`Update`].
+// TODO: Currently this type is directly copied from BDK  for now->
+// but check that whether these `keychainkind` & `Anchor trait impl` are good to go with.
+pub type ChangeSet = bdk_persist::CombinedChangeSet<KeychainKind, ConfirmationTimeHeightAnchor>;
+
+pub(crate) type SecpCtx = bdk_Secp256k1<All>;
```

Comment:
This feels redundant. Just initialize a secp context at the call site whenever it's required.